Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridColumnHeadersPresenter.cs

```diff
@@ -338,11 +338,11 @@ protected override Size MeasureOverride(Size availableSize)
                 {
                     columnHeader.Measure(new Size(column.ActualMaxWidth, double.PositiveInfinity));
                     this.OwningGrid.AutoSizeColumn(column, columnHeader.DesiredSize.Width);
-                    column.ComputeLayoutRoundedWidth(totalDisplayWidth);
+                    column.ComputeLayoutRoundedWidth(totalDisplayWidth, GetScale());
```

Comment:
>GetScale [](start = 72, length = 8)

Why pass in GetScale here, when it could be computed form OwningGrid in ComputeLayoutRoundedWidth?

Reply:
Addressed by https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3206/commits/74b7c128681b06c5bc65220eeea1716dacb6f6c5